File: java/src/main/java/bouchonnois/repository/PartieDeChasseRepository.java
package bouchonnois.repository;

import bouchonnois.domain.PartieDeChasse;

import java.util.UUID;

public interface PartieDeChasseRepository {
    void save(PartieDeChasse partieDeChasse);

    PartieDeChasse getById(UUID partieDeChasseId);
}

File: kotlin/refactoring-du-bouchonnois/src/main/kotlin/bouchonnois/repository/PartieDeChasseRepository.kt
package bouchonnois.repository

import bouchonnois.domain.PartieDeChasse
import java.util.*

interface PartieDeChasseRepository {
    fun save(partieDeChasse: PartieDeChasse)
    fun getById(partieDeChasseId: UUID?): PartieDeChasse?
}